### ISSUE TITLE
Advanced HTML and CSS Course: Add an additional resource to The Web Content Accessibility Guidelines (WCAG)

### ISSUE THREAD

**Issue by Brandie-M** [TheOdinProject/curriculum#27971]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

The additional resources page mentions it is seeking additional resources. 

> This section contains helpful links to related content. It isn’t required, so consider it supplemental.
> 
> -   It looks like this lesson doesn’t have any additional resources yet. Help us expand this section by contributing to our curriculum.

Suggested additional resources: 

> This section contains helpful links to related content. It isn’t required, so consider it supplemental.
> 
>    - [The A11Y Project:](https://www.a11yproject.com/) An open-source resource that provides information on how to make your web content more accessible. The A11Y Project includes checklists, guidelines, and tools that can help you implement WCAG standards effectively.
> - [Introduction to Web Accessibility by W3Cx (edX](https://www.edx.org/learn/web-accessibility/the-world-wide-web-consortium-w3c-introduction-to-web-accessibility)): This free course, created by the World Wide Web Consortium (W3C) and offered on edX, provides an introduction to web accessibility. It covers the principles of WCAG and practical steps for creating accessible web content.

### Path

Ruby / Rails, Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/advanced-html-and-css-the-web-content-accessibility-guidelines-wcag

**Comment by CouchofTomato** [comment]
@TheOdinProject/html-css Can someone give their opinion on this please.

**Comment by MaoShizhong** [comment]
I like The A11Y Project as an additional resource.

I feel the edx course is a bit much, given it's a "4-week" course (self paced yes, but that's still representative of what they deem appropriate for their course content). I also am not the biggest fan of edx's approach with many courses where there's a free track and paid track, where they limit access to some things in the free track. Especially when exactly what is limited varies from course to course.

I'd be in favour of adding The A11Y project as an additional resource, without the edx course. Since you specified you didn't want to work on this, I'll open this up for assignment.

## Acceptance criteria

Comment below if you'd like to be assigned to this issue. Please do not open a PR unless you have been assigned by a maintainer.

- [ ] Add [The A11Y Project](https://www.a11yproject.com/) as an additional resource to our [Web Accessibility lesson](https://www.theodinproject.com/lessons/advanced-html-and-css-the-web-content-accessibility-guidelines-wcag)

**Comment by kellyky** [comment]
I'd like to pick this up, @MaoShizhong
